File: Napakalakirb/lib/prize.rb
#encoding:utf-8
# To change this license header, choose License Headers in Project Properties.
# To change this template file, choose Tools | Templates
# and open the template in the editor.

require_relative "treasure.rb"
require_relative "treasure_kind.rb"

class Prize
  attr_reader :treasures
  attr_reader :level
  
  def initialize(treasures, level)
    @treasures = treasures
    @level = level
  end
  
  def to_s
    "Tesoros ganados: #{@treasures}, Niveles ganados: #{@level}"
  end
  
end

File: Napakalaki/src/napakalaki/Prize.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package napakalaki;

/**
 *
 * @author jose
 */
public class Prize {
    private int treasures , level ;
    public Prize( int treasures , int level ){
        this.treasures = treasures ;
        this.level = level ;
    }
    public int getTreasures (){
        return treasures;
    }
    public int getLevel(){
        return level;
    }
    public String toString(){
        return "Tesoros ganados: " + Integer.toString(treasures) + " Niveles ganados: " + Integer.toString(level);
    }
}
